account.go

```diff
@@ -12,8 +13,12 @@ func (c *Client) AddAccount(ctx context.Context, account model.Account, opts ...
 		return err
 	}
 	// TODO: Account →　FBS
-	res, err := client.Torii(ctx, nil, opts...)
-	c.Logger.Printf("[INFO]\tAddAccount\tMessage\t%s", res.Message())
+	builder := flatbuffers.NewBuilder(0)
+	a := account.Serialize(builder)
+	builder.Finish(a)
+
+	_, err = client.Torii(ctx, builder, opts...)
+	//c.Logger.Printf("[INFO]\tAddAccount\tMessage\t%s", res.Message())
```

Comment:
If you refer to the `res.Message()` here, you will get an error, so I commented out